pennylane/labs/vibrational/christiansen_utils.py

```diff
@@ -848,7 +848,7 @@ def christiansen_integrals_dipole(pes, n_states=16):
     comm.Barrier()
     dipole_cform_onebody = comm.bcast(dipole_cform_onebody, root=0)
 
-    if pes.localized is True or pes.dipole_level > 1:
+    if pes.dipole_level > 1:
         local_dipole_cform_twobody = _cform_twomode_dipole(pes, n_states)
```

Comment:
So to confirm `_cform_twomode_dipole` and `_cform_threemode_dipole` do not use any approximations based on the localization of the PES?

Reply:
No, it should not be influenced by that.